README.md

```diff
@@ -2,6 +2,43 @@
 
 Automate onboarding with https://github.com/packit/dist-git-to-source-git
 
+## Decommissioning
+
+**As of Q1 2022, we are planning to decommission the `c8` branches and repositories.**
```

Comment:
Won't this also include `c8s` branches? I think those were also created when CentOS dist-git had them, `c8` ones were just used as a fallback.

Reply:
oh, this is a typo, since there are c8, c8s, c8s-stream-$module_version - I'll explain and replace c8 with c8*

thanks for pointing this out